src/app/util/types_stub.h

```diff
@@ -175,61 +175,22 @@ typedef struct
 /**
  * @brief The broadcast endpoint, as defined in the ZigBee spec.
  */
-#define EMBER_BROADCAST_ENDPOINT (chip::kInvalidEndpointId)
+#define MATTER_BROADCAST_ENDPOINT (chip::kInvalidEndpointId)
```

Comment:
It doesn't make sense to have a MATTER_BROADCAST_ENDPOINT "as defined in the ZigBee spec".

I suggest removing this thing entirely and just using kInvalidEndpointId in the few places it's used.

Reply:
removed